test/handlers_test.go

```diff
@@ -18,7 +18,9 @@ func Test_ValidateDeployRequest_ValidCharacters(t *testing.T) {
 		{"lower", "abz"},
 		{"upper", "ABZ"},
 		{"upper and lower mixed", "AbZ"},
-		{"includes dashes", "test-function"},
+		{"includes hyphen", "test-function"},
+		{"can start with a digit", "1AbZ"},
+		{"can end with a digit", "AbZ1"},
```

Comment:
hmm.. the regex says no capital letters I think? So these should now be invalid

Reply:
@alexellis I'd fixed that locally and forgot to do `git add` :joy: